flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/sdk/OpenedUpdates.kt

```diff
@@ -0,0 +1,21 @@
+package flank.scripts.shell.firebase.sdk
+
+import com.github.kittinunf.result.getOrElse
+import com.github.kittinunf.result.onError
+import flank.scripts.github.getGitHubIssueList
+
+suspend fun checkForOpenedUpdates(token: String) = getGitHubIssueList(
+    githubToken = token,
+    parameters = listOf(
+        "creator" to "github-actions[bot]",
+        "state" to "open",
+        "labels" to "gcloud SDK"
+    )
+)
```

Comment:
odd spacing here
Feels like there is one new line too many

Reply:
`ktlint` formatted that way.